homepage.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a6627283f4.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="top-bg">
+        <div>
+        
+            <ul class="top">
+                <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+                <a href=""><i class="fa-brands fa-twitter"></i></a>
+              <li>Logout</li>
+              <li>Contact</li>
+              <li>Kenya</li>
+            </ul>
+        </div>
+    </nav>
+    <header class="header">
+        <div class="lg-logo">
+            <a href="index.html">
+              <img src="Images/compost_FILL.svg" alt="logo img">
+            </a>
+        <a  class="logo" href="">Eco Tourism<br> Kenya 2022 </a>
+        </div>
+    
+            <nav class="nav-1">
+            <ul class="list">
+                <a href="">
+                    <li>Home</li>
+                </a>
+                <a href="index.html">
+                    <li>About</li>
+                </a>
+                <a href="">
+                    <li>Sponsor</li>
+                </a>
+                <a href="">
+                    <li>News</li>
+                </a>
+                <a href=""><li class="btn" type="button">Campaign</li></a>
+            </ul>
+            <div class="menu"> 
+                <span class="bar-2"></span>
+                <span class="bar-2"></span>
+                <span class="bar-2"></span>
+            </div>
+        </nav>
+    </header>
+<section>
+    <div class="about-section">
+        <h2 class="sub-abt-h">ISIOLO DROUGHT <br> RELIEF RESPONSE <br> 2022</h2>
+       
+        <div>
+            <p class="about-pr">
+                Drought is already affecting all types of countries, in all types of geographies,<br>
+                 negatively impacting global food systems, health, economic and social development.<br>
+                  This trend is likely to continue unless we all get behind the necessary change and invest in solutions.<br>
+                   Insufficient rainfall is not the only cause of drought.<br> 
+                There is a lack of awareness about human-induced water scarcity that exacerbates drought risks and effects. 
+            </p>
+            <p class="about-address">
+                Please contact us for awareness and how to fight crime.UN Nations 2022.<br>
+                unnationorg.gmail.com
+            </p>
+        </div>
```

Comment:
Dear @monicah79, As the previous code reviewer mentioned please wrap the articles inside an article tag. this is for semantic reasons.